addons/crm_addons/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'CRM Add-ons',
+    'version': '1.0',
+    'category': 'Sales/CRM',
+    'sequence': 5,
+    'summary': 'Services to connect web add-ons to the CRM',
+    'description': "",
```

Comment:
What about :

> This module allows web add-ons to retrieve data from the CRM module such as contact information and leads, and do some operation such as automatically creating and completing new contacts, creating leads, and posting the content of the mails to the chatter.

This is a separate module, because this API should not automatically be exposed with every CRM installation, and should be upgradable independently.